default/stringtables/es.txt

```diff
@@ -1167,11 +1167,45 @@ Escalado de coste de partes
 #*RULE_TEST_STRING_DESC
 #* <not translated 1970-01-01 01:00:00>
 
-#*RULE_STARLANES_EVERYWHERE
-#* <not translated 1970-01-01 01:00:00>
+RULE_STARLANES_EVERYWHERE
+Vías Estelares Completas
 
-#*RULE_STARLANES_EVERYWHERE_DESC
-#* <not translated 1970-01-01 01:00:00>
+RULE_STARLANES_EVERYWHERE_DESC
+Se crean vías estelares entre cada par de sistemas, independientemente de la geometría. Estas vías pueden, aún así, ser eliminada durante la partida. Estas vías no aparecen en el mapa de la galaxia.
+
+RULE_ALL_OBJECTS_VISIBLE
+Todos los objetos visibles
+
+RULE_ALL_OBJECTS_VISIBLE_DESC
+Todo sistema, planeta, edificio, flota, nave o campo puede ser detectado por todos los imperios con al menos visibilidad parcial.
+
+RULE_ALL_SYSTEMS_VISIBLE
+Todos los sismteas visibles
+
+RULE_ALL_SYSTEMS_VISIBLE_DESC
+La localización de todos los sistemas es conocida por todos los imperios.
+
+RULE_EXTRASOLAR_SHIP_DETECTION
+Detección de naves extrasolares
+
+RULE_EXTRASOLAR_SHIP_DETECTION_DESC
+Las naves fuera del sistema pueden otorgan capacidad de detección
```

Comment:
I like this more:
"Las naves fuera de sistemas solares tienen visión"
It is more explicative for casual players than the English "provide detection" (provide who with detection? the ship provides itself with detection).
If you prefer to be stick to a more direct translation, it should be this:
"Las naves fuera de sistemas solares otorgan detección" (_sistemas_ in plura)